gdpr/vendorlist-fetching.go

```diff
@@ -11,35 +11,37 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/prebid/go-gdpr/api"
 	"github.com/prebid/go-gdpr/vendorlist"
+	"github.com/prebid/go-gdpr/vendorlist2"
 	"github.com/prebid/prebid-server/config"
 	"golang.org/x/net/context/ctxhttp"
 )
 
-type saveVendors func(uint16, vendorlist.VendorList)
+type saveVendors func(uint16, api.VendorList)
 
 // This file provides the vendorlist-fetching function for Prebid Server.
 //
 // For more info, see https://github.com/prebid/prebid-server/issues/504
 //
 // Nothing in this file is exported. Public APIs can be found in gdpr.go
 
-func newVendorListFetcher(initCtx context.Context, cfg config.GDPR, client *http.Client, urlMaker func(uint16) string) func(ctx context.Context, id uint16) (vendorlist.VendorList, error) {
+func newVendorListFetcher(initCtx context.Context, cfg config.GDPR, client *http.Client, urlMaker func(uint16, int) string, tCFVer int) func(ctx context.Context, id uint16) (vendorlist.VendorList, error) {
```

Comment:
Could we just name it TCFVer here? .. since capitalization doesn't affect method parameters.

Reply:
Done